content/en/blog/2024/new-otel-features-envoy-istio/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: 'Observability on the Edge: Enhancing OTel support in Envoy and Istio'
+linkTitle: New OTel features in Envoy and Istio
+date: 2024-05-28
+author: '[Joao Grassi](https://github.com/joaopgrassi) (Dynatrace)'
+issue: 4534
+sig: OpenTelemetry Specification
+---
+
+In the dynamic world of cloud-native and distributed applications, managing
+microservices effectively is critical. Kubernetes has become the de facto
+standard for container orchestration, enabling seamless deployment, scaling, and
+management of containerized applications.
+
+The distributed nature of such systems, however, adds a layer of complexity in
+the form of networking for in-cluster communication. Two well-known projects,
+Envoy and Istio, have emerged as the foundation for the smooth management and
+operation of such complex environments.
+
+Together, these technologies empower organizations to build scalable, resilient,
+and secure distributed systems.
+
+[Istio](https://istio.io/) is a service mesh, that orchestrates communication
+between microservices, providing features such as traffic management, security
+and, of course observability. Istio uses the Envoy proxy as its data plane.
+[Envoy](https://www.envoyproxy.io/) is a high-performance proxy, designed for
```

Comment:
I prefer introducing the links to projects when they are mentioned for the first time (see my suggestion above), but this is also OK, since you also provide some details on what those projects are. Choose what you prefer.

Reply:
Yeah I was also in between the two options, but I prefer to link later together with the bit of explanation of what those are.